### DISCUSSION TITLE
Nuclei support for DISA STIGs and CIS Benchmarks

### DISCUSSION THREAD

**Discussion by floydrodery** [projectdiscovery/nuclei#3178] [Ideas]
Popular Enterprise products that are leveraged for vulnerability scans and management also include the ability to run a DISA STIG or CIS Benchmark against an asset to check for "best practices" regarding the configuration (not vuln related).

DISA STIGs are released in XCCDF (Extensible Configuration Checklist Description Format) and can be imported and executed against a target to automatically validate the OS or application configuration settings.  Outside of the commercial enterprise products like Nessus, can this be accomplished with Nuclei?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
